content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -160,6 +160,15 @@ instance types and preferences that users can select later. [Read how to add new
 
 ### Virtual Machine Scheduling
 
+KubeVirt can take advantage of Kubernetes inner features to provide an advanced scheduling mechanism to virtual machines (VMs):
+- [Kubernetes topology spread constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/)
+- [Kubernetes node affinity/anti-affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity)
+
+Since KubeVirt VMs are wrapped up in Pods, the Kubernetes scheduling rules applying to Pods are completely valid to KubeVirt VMs.
```

Comment:
Please change to something like that
> Since KubeVirt VMs are wrapped in pods, the Kubernetes scheduling rules applicable to pods are completely valid for KubeVirt VMs.